### ISSUE TITLE
Install with PacletInstall

### ISSUE THREAD

**Issue by oameye** [rokzitko/sneg#1]
Would be handy, if the package could provide a handy install command with `PacletInstall`. Most Mathematica packages are moving to this system. For example for [NCAlgebra](https://github.com/NCAlgebra/NC/tree/v6.0.0) newest release or [MaTeX](http://szhorvat.net/pelican/latex-typesetting-in-mathematica.html):
```
PacletInstall["https://github.com/NCAlgebra/NC/blob/v6.0.0/NCAlgebra-6.0.0.paclet?raw=true"];

ResourceFunction["MaTeXInstall"][]
```

I am willing to find out what is needed for this.

**Comment by rokzitko** [comment]
Yes, that would be useful. Alternatively (or in parallel), it would be nice if it could be installed from GUI through File/Install menu action. I'll look into this when I find some time, but if you can point me to where these things are documented, that would be great.

(In practice, what I actually prefer to do on my computers is to make a symlink from Mathematica Applications dir to a cloned github repo..)

**Comment by oameye** [comment]
The functionality to install the `Sneg` package through a GUI File/Install menu already is available. Could be nice if this is documented on the website and here on GitHub. 
![NfxVsMiOEx](https://user-images.githubusercontent.com/57623933/222980122-b7117961-3907-4838-ae9c-773918bdf0a2.gif)

I also looked up some tutorials on how to make a Paclet available:
*  [Mathematica documentation](https://reference.wolfram.com/language/PacletTools/tutorial/CreatingPaclets.html)
* [Adding a Paclet to the Mathematica Paclet Server](https://b3m2a1.github.io/adding-a-paclet-to-the-mathematica-paclet-server#body)
* [Creating a Mathematica Package Repository](https://b3m2a1.github.io/creating-a-mathematica-package-repository#body)
* [Making a Paclet with BTools](https://b3m2a1.github.io/making-a-paclet-with-btools#body)
* [Building a Mathematica Package Ecosystem](https://b3m2a1.github.io/building-a-mathematica-package-ecosystem-part-1.html)

Some todo's to close this issue would be:
* [x] Add the functionality to install the package through Mathematica GUI to the documentation
* [ ] Make a Paclet and host it on GitHub such that the package easily can be installed with the `PacletInstall[[]` function.

**Comment by rokzitko** [comment]
Thank you. I'll look into Paclets when I find some time.

**Comment by oameye** [comment]
No worries. I would do it too, but it seems that there are different decisions one could take. So I think it is best that the package Owner makes those.

Ps: I really like the package and open to contribute ones I have looked at the code a bit.
